File: backend-dotnet/Data/Daos/Projects.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

[Table("projects", Schema = "taskmanager")]
public class Projects
{
	public Projects()
	{
	}

	[Key]
	[Column("pk")]
	public long Pk {  get; set; }
	[Column("project_id")]
	public string ProjectId {  get; set; }
	[Column("name")]
	public string Name {  get; set; }
	[Column("description")]
	public string? Description { get; set; }
	[Column("created_on")]
	public DateOnly CreatedOn { get; set; }
}

File: backend-python/daos/projects.py
from sqlalchemy import Column, String, BigInteger, Date 
from database import Base

class Projects(Base):
    __tablename__ = "projects"
    # Only if you want to override schema
    # __table_args__ = {"schema": "taskmanager"}

    pk = Column(BigInteger, primary_key= True, autoincrement = True)
    project_id = Column(String(80), nullable= False)
    name = Column(String(120), nullable = False)
    description = Column(String, nullable = True)
    created_on = Column(Date, nullable = False)